pkg/storage/store.go

```diff
@@ -15,10 +16,13 @@ const MaxObjectSizeInMemory = 2 * 1024 * 1024 * 1024 // 2 gigs
 func (e errString) Error() string                    { return string(e) }
 
 const (
-	IfNotPresent = true
-	OverWrite    = false
+	// Adding these to make code more readable when looking at Put Call
+	NoOverWrite = true
+	OverWrite   = false
 )
 
+type NewKey bool
```

Comment:
declaring NewKey as a type alias instead of a full-fledged type Would make conversions unnecessary.
`type NewKey = bool`

Reply:
Thank you!